galbi/cli.py

```diff
@@ -180,28 +183,87 @@ def get(key: typing.List[str]):
             break
         if d['title'] in keys:
             keys.remove(d['title'])
-            latest_comment = None
-            for comment in fetch_all_pages(
-                lambda page, per_page: http.get(
-                    d['comments_url'],
-                    params={
-                        'per_page': per_page,
-                        'page': page,
-                    }
-                )
-            ):
-                latest_comment = comment
-            else:
-                cache[d['title']] = json.loads(
-                    latest_comment['body']
-                )
+            cache[d['title']] = json.loads(d['body'])
     echo(json.dumps(cache, indent=4))
 
 
+@command()
+@option('--key', '-k')
+@option('--rev', '-r', type=int)
+def get_rev(key: str, rev: int):
+    config = load_config(default_config_json)
+    http = get_http_session(default_config_json)
+    repo = config['repo']
+    resp = http.get(
+        urllib.parse.urljoin(github_api_url, f'/repos/{repo}/issues'),
+        params={
+            'state': 'open',
+            'per_page': 1,
+            'labels': [key],
+        }
+    )
+    resp.raise_for_status()
+    payload = resp.json()[0]
+    page = rev // 100
+    resp = http.get(
+        payload['comments_url'],
+        params={'per_page': rev % 100 + 1, 'page': page}
+    )
+    resp.raise_for_status()
+    echo(json.dumps(json.loads(resp.json()[-1]['body']), indent=2))
```

Comment:
Shouldn't `-1` be `rev % 100`?

Reply:
nope, `-1` is last element of list.

```
>>> [1,2,3]
>>> _[-1]
3
```

Reply:
i misunderstood, https://github.com/hopsoffice/galbi/pull/4/files#diff-cd645e754bb0b3cd5c77307f87ee59dacb76dcae72e43b95887cfcc5fb23b7f8R207-R213 fixed it

Reply:
this link is more readable: https://github.com/hopsoffice/galbi/compare/12c05a9e91cde73141b6971b3574e331e5b964da..7fda10c435d452e8a62f67587301e1a5a4f9d9cf